cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/BridgeConfiguration.java

```diff
@@ -23,15 +23,20 @@
     private Collection<ProxyFallbackConfiguration> bungeeFallbackConfigurations;
     private Map<String, String> messages;
     private boolean logPlayerConnections = true;
+    private boolean hubCommandEnabled = true;
+    private ArrayList<String> hubCommandAliases;
 
     public BridgeConfiguration(String prefix, boolean onlyProxyProtection, Collection<String> excludedOnlyProxyWalkableGroups, Collection<String> excludedGroups,
-                               Collection<ProxyFallbackConfiguration> bungeeFallbackConfigurations, Map<String, String> messages, boolean logPlayerConnections) {
+                               Collection<ProxyFallbackConfiguration> bungeeFallbackConfigurations, Map<String, String> messages,
+                               boolean logPlayerConnections, boolean hubCommandEnabled, ArrayList<String> hubCommandAliases) {
         this.prefix = prefix;
         this.onlyProxyProtection = onlyProxyProtection;
         this.excludedOnlyProxyWalkableGroups = excludedOnlyProxyWalkableGroups;
         this.excludedGroups = excludedGroups;
         this.bungeeFallbackConfigurations = bungeeFallbackConfigurations;
         this.messages = messages;
+        this.hubCommandEnabled = hubCommandEnabled;
```

Comment:
Maybe instead of an extra boolen if the aliases are empty? @derrop @juliarn ?

Reply:
Personally I like the option with an extra boolean more. Maybe this guarantees fewer problems with the user than telling him that he has to remove everything in the array for disabling the command.

Reply:
As you can see rop and julian have the same opinion, I would also prefer that to bool